aim/agent/aid/universes/aci/converter.py

```diff
@@ -351,13 +349,11 @@ def fv_rs_dom_att_converter(object_dict, otype, helper,
     'fvRsProv': [{'resource': resource.EndpointGroup,
                   'converter': fvRsProv_converter},
                  {'resource': resource.ExternalNetwork,
-                  'converter': fvRsProv_Ext_converter,
-                  'convert_pre_existing': True}],
+                  'converter': fvRsProv_Ext_converter}],
     'fvRsCons': [{'resource': resource.EndpointGroup,
                   'converter': fvRsCons_converter},
                  {'resource': resource.ExternalNetwork,
-                  'converter': fvRsCons_Ext_converter,
-                  'convert_pre_existing': True}],
```

Comment:
I think we still need to make a distinction between relations that we want to configure and others that we want to monitor